demo-shell/src/app/components/form/form.component.scss

```diff
@@ -32,7 +32,7 @@
 }
 
 .app-form-config-editor {
-    height: 500px !important;
+    min-height: 500px;
```

Comment:
min-height and height are different properties, and different purpose

Reply:
accommodated change with height property.